src/XModem.cpp

```diff
@@ -168,7 +170,9 @@ bool XModem::find_header() {
   } while(i++ < retry_limit);
   return false;
 }
-
+/**
+ * Equals bool _xmodem_rx(int *fd, struct xmodem_config *config) <-- linux
```

Comment:
This is unnecessary and more likely to go stale since it's referencing a method in a different platform implementation